ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15-periodics.yaml

```diff
@@ -8324,18 +8324,7 @@ periodics:
     ci.openshift.io/generator: prowgen
     job-release: "4.15"
     pj-rehearse.openshift.io/can-be-rehearsed: "true"
-  name: periodic-ci-openshift-openshift-tests-private-release-4.15-amd64-nightly-aws-ipi-ovn-hypershift-arm-nodepool-guest-f14-destructive
-  reporter_config:
```

Comment:
Could you keep the reporter without removing it?

Reply:
Good catch, I do not manually remove it I guess when I rebase solve the conflicts and make jobs again it deleted by mistake.

Reply:
Added back, also added the 4.18 missed reporter.